modules/mobile/docs/openapi.yaml

```diff
@@ -2435,6 +2435,78 @@ paths:
       security:
         - Bearer: []
       summary: /v0/messaging/health/folders/{id}/messages
+  /v0/messaging/health/folders/{id}/threads:
+    get:
+      description: List of threads in a secure messaging folder
+      operationId: get_folder_threads
+      parameters:
+        - description: The id of the folder that threads are being retrieved from
+          in: path
+          name: folderId
+          required: true
+          schema:
+            type: string
+        - description: The size of the pagination you want
+          in: query
+          name: pageSize,
+          required: true
+          schema:
+            type: string
+        - description: The page number to get based on your page size
+          in: query
+          name: page,
+          required: true
+          schema:
+            type: string
+        - description: The field to sort the results by
+          in: query
+          name: sortField,
+          required: true
```

Comment:
Do we know if all of these are actually required? Will our front end even be using them? The client is written to always use them but it's not clear if omitting them causes any issues because they don't have any sample specs addressing it, nor do they require the params in the controller. I guess we'll find out in testing.

Reply:
I can always set a default in the controller so that it's not required

Reply:
None of the fields outside of folder id are required so I required from those